src/OrbitGl/TrackManager.cpp

```diff
@@ -468,6 +475,20 @@ CGroupAndProcessMemoryTrack* TrackManager::CreateAndGetCGroupAndProcessMemoryTra
   return cgroup_and_process_memory_track_.get();
 }
 
+PagefaultTrack* TrackManager::CreateAndGetPagefaultTrack(
+    const std::array<std::string, orbit_gl::kBasicPagefaultTrackDimension>& series_names) {
+  std::lock_guard<std::recursive_mutex> lock(mutex_);
+  if (pagefault_track_ == nullptr) {
+    constexpr uint8_t kTrackValueDecimalDigits = 0;
+    pagefault_track_ = std::make_shared<PagefaultTrack>(time_graph_, time_graph_, viewport_,
+                                                        layout_, series_names, capture_data_);
+    pagefault_track_->GetMajorPagefaultTrack()->SetNumberOfDecimalDigits(kTrackValueDecimalDigits);
```

Comment:
What about providing a method from the pagefault_track to set the number of digits in the subtracks? (the method could have 2 input values)

Reply:
Since we need the Get subtrack method anyway, what is the benefits of adding this additional method in the pagefault track?

Reply:
I don't want TrackManager and TimeGraph to know the existence of the subtracks. IMO, they should be managed internally for the PageFaultTrack. I'm not sure if we can get rid of this method in ProcessTimer function for now (but I want to erase all these functions in the future).

Reply:
Make sense. I will add methods like `PagefaultTrack::AddValuesForMajorPagefaultSubtrack` and `PagefaultTrack::SetNumberOfDecimalDigits` to avoid the association between `TrackManager` and subtracks.